src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -167,11 +166,11 @@ public static Optional<String> getIndividualPreferredHearingChannel(String heari
     }
 
     public static Optional<String> getIndividualInterpreterLanguage(HearingOptions hearingOptions) {
-        if (isTrue(hearingOptions.wantsSignLanguageInterpreter())) {
+        if (null != hearingOptions && isTrue(hearingOptions.wantsSignLanguageInterpreter())) {
             return getSignLanguage(hearingOptions)
                 .map(SignLanguage::getHmcReference);
         }
-        if (isYes(hearingOptions.getLanguageInterpreter())) {
+        if (null != hearingOptions && isYes(hearingOptions.getLanguageInterpreter())) {
```

Comment:
The null check is duplicated, it would be better to check before with a quick fail